x-pack/platform/plugins/shared/ml/public/application/explorer/annotation_timeline.tsx

```diff
@@ -79,7 +83,7 @@ export const AnnotationTimeline = <T extends { timestamp: number; end_timestamp?
         .attr('y', 0)
         .attr('height', ANNOTATION_CONTAINER_HEIGHT)
         .attr('width', endingXPos - startingXPos)
-        .style('stroke', euiTheme.euiBorderColor)
+        .style('stroke', euiTheme.colors.borderBasePlain)
```

Comment:
Question: `euiBorderColor` would map to `euiTheme.border.color` which is `borderBaseSubdued` in Borealis.
Should this border be stronger here or can it be the regular `euiTheme.border.color`? (same for following usages)

Reply:
👍 I'm using `euiTheme.border.color` now to match the original one ([904b773](https://github.com/elastic/kibana/pull/203518/commits/904b7739669a5e59cf861cd64c11dcfdef20ae33)).